test/integration/content-explorer/ContentExplorer.e2e.test.js

```diff
@@ -0,0 +1,207 @@
+// <reference types="Cypress" />
+const helpers = {
+    load(additionalProps = { canUpload: true, permissions: { canUpload: true } }) {
+        cy.visit('/Elements/ContentExplorer', {
+            onBeforeLoad: contentWindow => {
+                contentWindow.PROPS = additionalProps;
+            },
+        });
+    },
+    getRow(rowNum) {
+        return cy.getByDataPreview('ContentExplorer').find(`.bce-item-row-${rowNum}`);
```

Comment:
A Cypress best practice we've been trying to follow is to use data-testid attributes to select elements of interests in our tests. You could do the same here and instrument the code with data-testid attribute for each row and then use `cy.getByTestId` like we do in other tests

Reply:
The rows are children of a react-virtualized table, and the table doesn't expose the html attributes of its individual rows, so I don't think I can add a data-testid attribute for the rows.  (I will add the data-testid attributes for the other components that we have control over.)